packages/BaselineOfMarkdownEditor.package/BaselineOfMarkdownEditor.class/instance/baseline..st

```diff
@@ -5,7 +5,12 @@ baseline: spec
 		for: #'common'
 		do: [
 			spec
-				package: 'MarkdownEditor-Core';
+				package: 'VB-Regex' with: [
+					spec repository: 'http://www.squeaksource.com/Regex'].
+			spec
+				package: 'MarkdownEditor-Core' with: [
+					(#('Squeak5.0') includes: Smalltalk version)
```

Comment:
(Question migrated from Telegram)
As we told smalltalkCI to only include our tests here:
https://github.com/hpi-swa-teaching/MarkdownEditor/blob/4cb5903f40b34b17f11993cbbe405929c3a15f65/.smalltalk.ston#L14

Do we still need the `#('Squeak5.0') includes: Smalltalk version`?

Otherwise, is this ready to be merged?

Reply:
Well, it works, but overwriting the post 5.1 Regex would be fatal in my opinion.

Reply:
As far as I understand baselines, you define dependencies here (similiar to a package manager). Then how would we overwrite the `VB-Regex` package when we just say "we are depending on it"?
After a quick search, AutoTDD, Vivide, and Widgets do also have the Regex package in their baseline; AutoTDD without version switch, the other with. I remember that we asked how to isolate their tests from ours, but not whether we can remove the version switch. So I guess we have to ask again...

Reply:
Yeah I know, these should be dependencies. But when they were only dependencies, why should the tests have been failed for 5.1+? This means the package must have been loaded into the image (despite the fact, the dependency is already fulfilled), don't they?

Reply:
You are correct, `VB-Regex` is in 5.1+ a patched version which we would overwrite. Leaving it as it is